.github/workflows/setup-quantic-examples-community.yml

```diff
@@ -0,0 +1,49 @@
+name: Setup Quantic Examples Community
```

Comment:
We are setting  two scratch orgs with two communities, so either just `Setup Quantic Examples` or `Setup Quantic Examples Communities`